particle/particle/convert.py

```diff
@@ -272,6 +274,9 @@ def update_from_mcd(full_table, update_table):
     update_table_neg.index = -update_table_neg.index
     full_table.update(update_table_neg)
 
+    # Only keep rows present in the update table, i.e. the .mcd file
+    full_table = full_table[full_table.index.isin(update_table.index) | full_table.index.isin(update_table_neg.index)]
```

Comment:
This looks fine for now.

Reply:
Yeah, otherwise we were providing particles with semi-OK properties more than a decade old, sometimes with best guesses. That's not great, I reckon. Better just not provide those particles.